packages/core/src/lib/androidSdk/AndroidSdkTools.ts

```diff
@@ -20,6 +20,7 @@ import util = require('../util');
 import {Config} from '../Config';
 import {JdkHelper} from '../jdk/JdkHelper';
 import {Log, ConsoleLog} from '../../lib/Log';
+import {Result} from '../..';
```

Comment:
Prefer explicit imports when importing via path.

```suggestion
import {Result} from '../../lib/Result';
```